src/components/AddFolderModal/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Button, Form, Input, Modal } from 'antd';
+
+import './styles.scss';
+
+function AddFolderModal({ handleCancel, isModalVisible }) {
```

Comment:
event function props should be named `on-`. on this case this should be `onCancel`